### PR TITLE
Update README installation/contributing section

### LINKED ISSUE #7: add contributing documentation
copy most of section contributing and developing a feature https://github.com/WomenWhoCode/WomenWhoCode#contributing-and-developing-a-feature

### PR DESCRIPTION
**Pull request by littleforest**
Adds instructions for creating and maintaining a fork of the repo.

Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,19 @@ This is the API for the WomenWhoCode web app. It works in conjunction with the w
...
 4. Develop your feature on your branch
 5. Add the files you changed: `git add file_name` (avoid using `git add .`)
 6. Commit your changes: `git commit -m "Message briefly explaining the feature"`
-7. We have small Pull Requests, try to keep one commit per feature. If you forgot to add changes, you can edit the previous commit: `git commit --amend`
-8. Push to your repo: `git push origin branch-name`
-9. Go into the Github repo and create a pull request explaining your changes
-10. You will need to add a message on the pull request notifying your changes to your reviewer
+7. We have small Pull Requests; try to keep one commit per feature.
+8. Push to your fork: `git push origin branch_name`
+9. Go into the wwcode_api GitHub repo and create a pull request explaining your changes
```
Comment (zassmin) [inline README.md]: can you add a blurb about linking issue to PR using one of github's keywords so the issue auto closes?https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by zassmin** [comment]
@littleforest ty! one little note, otherwise 👍 🙇

**Comment by littleforest** [comment]
@zassmin, I added some verbiage on how to auto-close an issue. Please review, thanks!

**Comment by zassmin** [comment]
👍 🎉 @littleforest

**Pull request merged**